backend/de.metas.acct.base/src/main/java/de/metas/acct/gljournal_sap/service/SAPGLJournalLoaderAndSaver.java

```diff
@@ -367,4 +379,38 @@ public DocStatus getDocStatus(final SAPGLJournalId glJournalId)
 				.map(headerRecord -> DocStatus.ofNullableCodeOrUnknown(headerRecord.getDocStatus()))
 				.orElse(DocStatus.Unknown);
 	}
+
+	@NonNull
+	public SAPGLJournal createHeader(@NonNull final SAPGLJournalCreateRequest sapglJournalCreateRequest)
```

Comment:
imho u shall call this method `create` and do everytihng here.

there is no point do divide the logic of creation between this class and repository.

in repository u will just delegate the call to this method.

Reply:
also, consider making it package level is possible